i18n/README.pt-br.md

```diff
@@ -182,7 +174,7 @@ Nossa biblioteca de cliente é modular. Cada sub-biblioteca é uma implementaç
     <td>-</td>
   </tr>
   <tr>
-    <td>Rust</td>
+    <td>Ferrugem</td>
```

Comment:
@lyqht Ideally this would have remained as `Rust` in all languages. Does DeepL support any kind of word tagging to prevent this?